packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -81,7 +81,7 @@ export enum TemplateNames {
   CommandAndResponse = "command-and-response",
   Workflow = "workflow",
   DefaultBot = "default-bot",
-  MessageExtension = "message-extension",
+  MessageExtension = "message-extension-action",
```

Comment:
How about renaming this variable to "MessageExtensionAction" to avoid confusion.

Reply:
Sure, will fix.